tests/integration/common/s2n_test_common.py

```diff
@@ -0,0 +1,135 @@
+##
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#  http://aws.amazon.com/apache2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+#
+
+"""
+Common functions to run s2n integration tests.
+"""
+
+import subprocess
+
+from common.s2n_test_scenario import Mode, Version, run_scenarios
```

Comment:
Not blocking: this is an absolute import and is a little weird here. This module will only load if imported from the proper location, thus increasing fragility. However, the proper fix is restructuring the layout of the integration test folder. That seems out of scope for this issue. But something we could look at once functionality is complete.